src/cli/cli.cpp

```diff
@@ -2105,8 +2105,14 @@ otError Interpreter::ProcessIpMulticastAddrAdd(Arg aArgs[])
 {
     otError      error;
     otIp6Address address;
+    otIp6Address address2;
 
     SuccessOrExit(error = aArgs[0].ParseAsIp6Address(address));
+    if (!aArgs[1].IsEmpty())
```

Comment:
Can we support up to N addresses?

I suppose the code won't be too complex.

Reply:
Yes that sounds feasible. For CPU/memory efficiency one would use a best-effort registration, i.e. one of below
1. It registers each address 0...N until an address parse fails or an address registration fails.  That stops the process and remaining addresses are not treated.
2. It registers each address 0...N until the end and skips an entry if its address parse or address registration fails.

Reply:
Either works for me. 
Prefer 1 slightly though.

Reply:
Should we also output how many addresses was added successfully?

e.g. 
```
> ipmaddr add ff04::1 ff04::2 ff04:3
2 added
Error 6: Parse
```

This way users know exactly which address failed and which addresses were added successfully.

Reply:
The best-effort registration nr 1 is now included. The above output may be useful; but not included yet.